stories/demos/ColumnsReordering.tsx

```diff
@@ -66,15 +66,10 @@ function createColumns(): Column<Row>[] {
 export function ColumnsReordering() {
   const [rows] = useState(createRows);
   const [columns, setColumns] = useState(createColumns);
-  const [[sortColumn, sortDirection], setSort] = useState<[string, SortDirection]>([
-    'task',
-    'NONE'
+  const [sortColumns, setSortColumns] = useState<readonly Readonly<SortColumn>[]>([
+    { columnKey: 'task', direction: 'NONE' }
```

Comment:
When we have `direction: 'NONE'`, it's possible to add another sort column, but we still display the other's priority.
IMO we should just remove `NONE` as a possible sort direction. Just remove the `SortColumn` instead.
Sounds good @amanmahajan7?

![image](https://user-images.githubusercontent.com/567105/118993225-3cfa0180-b97d-11eb-823f-33ea057d5104.png)

Reply:
Sounds good to me